src/components/Kurlsh.js

```diff
@@ -1909,6 +1931,47 @@ class Kurlsh extends React.Component {
                 </div>
               </div>
 
+              <div className="flex flex-column u-marginTop--40">
+                <span className="u-fontSize--normal u-fontWeight--medium u-color--bermudaGray"> Cloud Provider </span>
+                <div className={`AddOn--wrapper ${selectedVersions.aws.version !== "None" && "selected"} flex flex-column u-marginTop--15`} onClick={(e) => this.handleIsAddOnSelected("aws", e)}>
+                  <div className="flex flex1">
+                    <div className="flex flex-auto alignItems--center">
+                      <input
+                        type="checkbox"
+                        className="u-marginRight--normal"
+                        checked={selectedVersions.aws.version !== "None"}
+                        readOnly
+                      />
+                      <span className="icon u-aws u-marginBottom--small" />
+                      <div className="flex flex-column u-marginLeft--15 u-marginTop--small">
+                        <div className="FormLabel"> AWS </div>
+                        <div className="flex flex1 alignItems--center">
+                          <div className={`SelectVersion flex flex1 ${!this.state.isAddOnChecked["aws"] && "disabled"}`} style={{ width: "200px" }}>
+                            <span className="flex alignItems--center u-color--fiord u-fontSize--normal versionLabel"> {!this.state.isAddOnChecked["aws"] ? "Version None" : "Version"} </span>
+                            <Select
+                              isSearchable={false}
+                              options={versions.aws}
+                              getOptionLabel={this.getLabel("aws")}
+                              getOptionValue={(aws) => aws}
+                              value={selectedVersions.aws}
+                              onChange={this.onVersionChange("aws")}
+                              matchProp="value"
+                              isDisabled={!this.state.isAddOnChecked["aws"]}
+                              isOptionSelected={() => false} />
```

Comment:
what is this always false?
```suggestion
                              isOptionSelected={() => false} />
```